submissions/OlStaniev/DOM/script/script.js

```diff
@@ -0,0 +1,46 @@
+import episodes from './text-episode.js';
+
+const menuList = document.querySelector('.nav__list');
+const introText = document.querySelector('.introText')
+const burgerIcon = document.querySelector('.header__burgerMenu')
+
+document.addEventListener('click', (event) => {
+    const episodeName = event.target.closest('.nav__list_item')
+    if (episodeName) {
+        const content = generateText(episodeName.dataset.id)
+        addContent(content,introText)
+    }
+    burgerMenuOpen(event)
+})
+
+function burgerMenuOpen(event) {
+    if (event.target.closest('.header__burgerMenu')) {
+        menuList.classList.toggle('nav__listOpen')
+        burgerIcon.classList.toggle('burgerCross')
+    }
+    if (!event.target.closest('.header__burgerMenu')) {
+        menuList.classList.remove('nav__listOpen')
+        burgerIcon.classList.remove('burgerCross')
+    }
+}
+
+function generateText(episodeId) {
+    const currentEpisode = episodes.find((episodeNumber) => episodeId === episodeNumber.id)
+    const storyText = `<div class="introText_wrapper"><h2>${currentEpisode.title}</h2><h3>${currentEpisode.subTitle}</h3><p>${currentEpisode.p1}</p><p>${currentEpisode.p2}</p><p>${currentEpisode.p3}</p></div>`
+    return storyText
```

Comment:
Also, use destructuring.
Write code more readable, not one line.
Get rid of the extra variable, just return the template.